### PR TITLE
Add newer Boost versions to Boost_ADDITIONAL_VERSIONS

### PR DESCRIPTION
**Pull request by SunBlack**

### CODE REVIEW

**Comment by UnaNancyOwen** [comment]
(This is a small thing,) It seems to be lined-up 5 versions on 1 line in basically.
Please move "1.70.0", "1.70" to new line. 
```
"1.70.0" "1.70"
"1.69.0" "1.69" "1.68.0" "1.68" "1.67.0" "1.67" "1.66.0" "1.66" "1.65.1" "1.65.0" "1.65"
```

**Comment by SunBlack** [comment]
> (This is a small thing,) It seems to be lined-up 5 versions on 1 line in basically.

I expected this comment 😆 But it is a bit strange to have a nearly empty line at beginning and not at last. Maybe we should change order of versions, or does this have any side effect?

Btw: If I move `"1.70.0" "1.70"` to a separate line, next line has still 6 entries, because of `"1.65.1"`.

**Comment by UnaNancyOwen** [comment]
> Maybe we should change order of versions, or does this have any side effect?

(This is my unfounded conjecture.) Maybe, I think that affects which version of Boost to find in case of installed multiple versions in system. What do you think?

> Btw: If I move "1.70.0" "1.70" to a separate line, next line has still 6 entries, because of "1.65.1".

I think that is an exception. :)

It is a very small problem whether add new line than change order.
It will not affect the operation. Which do you like new line or one line?

File: PCLConfig.cmake.in [hunk 1]
```diff
@@ -101,7 +101,8 @@ macro(find_boost)
   endif()
   set(Boost_ADDITIONAL_VERSIONS
     "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@.@Boost_SUBMINOR_VERSION@" "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@"
-    "1.68.0" "1.68" "1.67.0" "1.67" "1.66.0" "1.66" "1.65.1" "1.65.0" "1.65"
+    "1.70.0" "1.70"
+	"1.69.0" "1.69" "1.68.0" "1.68" "1.67.0" "1.67" "1.66.0" "1.66" "1.65.1" "1.65.0" "1.65"
```
Comment (UnaNancyOwen) [inline PCLConfig.cmake.in]: Please replace 1 tab to 4 space.

**Comment by UnaNancyOwen** [comment]
Thanks! 👍

**Pull request merged**